aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -80,13 +92,19 @@
     },
     "update": {
       "permissions": [
+        "sns:Publish",
+        "codeguru-profiler:AddNotificationChannels",
+        "codeguru-profiler:RemoveNotificationChannel",
         "codeguru-profiler:PutPermission",
         "codeguru-profiler:RemovePermission",
         "codeguru-profiler:GetPolicy"
       ]
     },
     "delete": {
       "permissions": [
+        "sns:Publish",
+        "codeguru-profiler:GetNotificationConfiguration",
+        "codeguru-profiler:RemoveNotificationChannel",
         "codeguru-profiler:DeleteProfilingGroup"
       ]
```

Comment:
This should not be needed -- AFAIK our API is built so that removing a profiling group cleans up all of its notification configurations (otherwise people may forget). Doublecheck if this really is needed.